webrender/src/device/gfx/device.rs

```diff
@@ -719,15 +716,16 @@ impl<B: hal::Backend> Device<B> {
                 .find(|format| format == &hal::format::Format::Bgra8Unorm)
                 .expect("Bgra8Unorm surface is not supported!")
         });
-
-        let mut extent = caps.current_extent.unwrap_or(hal::window::Extent2D {
-            width: (window_size.0 as u32)
-                .max(caps.extents.start.width)
-                .min(caps.extents.end.width),
-            height: (window_size.1 as u32)
-                .max(caps.extents.start.height)
-                .min(caps.extents.end.height),
-        });
+        let ext = (caps.current_extent.unwrap().width as i32, caps.current_extent.unwrap().height as i32);
```

Comment:
These `unwrap`s are dangerous here,  if we can't avoid them, at least use `expect` macro https://docs.rs/expect_macro/0.2.1/expect_macro/macro.expect.html like we do on other places.